kapitan/inputs/helm/template.go

```diff
@@ -0,0 +1,327 @@
+package main
+
+import "C"
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/Masterminds/sprig" // TODO added by copying vendor directory
+	"github.com/ghodss/yaml"
+	"io/ioutil"
+	"k8s.io/apimachinery/pkg/util/validation"
+	"k8s.io/helm/pkg/chartutil"
+	"k8s.io/helm/pkg/manifest"
+	"k8s.io/helm/pkg/proto/hapi/chart"
+	"k8s.io/helm/pkg/renderutil"
+	"k8s.io/helm/pkg/strvals"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"text/template"
+)
+
+const defaultDirectoryPermission_c = 0755
+
+type valueFiles []string
+
+func (v *valueFiles) String() string {
+	return fmt.Sprint(*v)
+}
+
+func (v *valueFiles) Type() string {
+	return "valueFiles"
+}
+
+func (v *valueFiles) Set(value string) error {
+	for _, filePath := range strings.Split(value, ",") {
+		*v = append(*v, filePath)
+	}
+	return nil
+}
+
+var (
+	whitespaceRegex_c = regexp.MustCompile(`^\s*$`)
+	// defaultKubeVersion is the default value of --kube-version flag
+	defaultKubeVersion_c = fmt.Sprintf("%s.%s", chartutil.DefaultKubeVersion.Major, chartutil.DefaultKubeVersion.Minor)
+)
+
+//readFile load a file from the local directory. remote file is disabled.
+func readFile(filePath string) ([]byte, error) {
+	return ioutil.ReadFile(filePath)
+}
+
+// Merges source and destination map, preferring values from the source map
+func mergeValues(dest map[string]interface{}, src map[string]interface{}) map[string]interface{} {
+	for k, v := range src {
+		// If the key doesn't exist already, then just set the key to that value
+		if _, exists := dest[k]; !exists {
```

Comment:
`exists` doesn't seem to be used after this if. Is there a point to assign `dest[k]` to it? 
Would `if !dest[k] { ` work?